src/components/form/form.styles.test.tsx

```diff
@@ -17,7 +17,7 @@ import {
   euiCustomControl,
 } from './form.styles';
 
-const darkModeWrapper: React.FC = ({ children }) => (
+const darkModeWrapper: React.FC<PropsWithChildren> = ({ children }) => (
```

Comment:
[super optional] Since we're already adding extra imports on line 9, we could as well grab `FunctionComponent` as well for consistency (this is on me btw, I wrote `React.FC` out of laziness when writing these tests originally 🙈)

```suggestion
const darkModeWrapper: FunctionComponent<PropsWithChildren> = ({ children }) => (
```

Reply:
Updated 👍🏻